pkg/web/filter.go

```diff
@@ -144,11 +140,12 @@ func (fs Filters) Chain(h Handler) Handler {
 		i := i
 		wrappedFilters[i] = HandlerFunc(func(request *Request) (*Response, error) {
 			params := map[string]interface{}{
-				"path":   request.URL.Path,
-				"method": request.Method,
+				"path":                 request.URL.Path,
+				"method":               request.Method,
+				log.FieldCorrelationID: CorrelationIDFromHeaders(request.Header),
```

Comment:
not sure if this is the best place for the correlation id generation - i think this is called as many times as fitlers are registered

Reply:
that's true.. we might remove it from here and we'll have a log "Entering filter" without correlation_id and "Exiting filter" with correlation id.

Reply:
Currently this implementation breaks the correlation ids if the incoming request doesn't have a correlation id header. This pkg/web/filter will generate one correlation id and the logging filter will generate another one which will be passed down